workbench-core/environments/src/services/environmentTypeService.ts

```diff
@@ -169,7 +169,7 @@ export default class EnvironmentTypeService {
       status: EnvironmentTypeStatus;
     }
   ): Promise<EnvironmentType> {
-    const id = uuidv4();
+    const id = uuidv4(); // id format will be updated as a part of GALI-1839
```

Comment:
still needed?

Reply:
I'll make this a TODO. But I do want to keep this in here to remind ourselves this should be updated.